share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
> However, I would still want someone to go back and check why info template is causing this issue when the title is too long?

It's actually part of the template. The subtitle is nested within the header tag. I had forgotten that's how it works though. We need to do some work bringing the same features to the templates. For example, the Text template doesn't allow a `url` but it very easily could.

You can view the template markup here: https://github.com/duckduckgo/duckduckgo-answerbar-templates/blob/master/detail/basic_info_detail.handlebars#L12